src/data/SongData.js

```diff
@@ -22,14 +23,13 @@ const songData = {
     user_role: "Singer",
     release_date: new Date(2017, 5).toString(),
     description: "J.Cole's best",
-    album_image:
-      "https://images.genius.com/6d0fbbc7ce189a8c81671ef92546446e.1000x1000x1.png",
+    album_image: "https://images.genius.com/6d0fbbc7ce189a8c81671ef92546446e.1000x1000x1.png",
     contributors: {
       1: { name: "John", role: "Producer", stake: 0.25 },
       2: { name: "Dan", role: "Sound engineer", stake: 0.25 },
-      3: { name: "J Cole", role: "Singer", stake: 0.5 },
+      3: { name: "J Cole", role: "Singer", stake: 0.5 }
```

Comment:
we can add trailing commas to our eslint file if we prefer that, but it'll probably apply it to every array and object

Reply:
I'm cool with whatever you prefer.